packages/ui-react/src/components/Card/Card.module.scss

```diff
@@ -154,23 +154,23 @@ $aspects: ((1, 1), (2, 1), (2, 3), (4, 3), (5, 3), (16, 9), (9, 16), (9, 13));
   text-align: left;
   text-shadow: 0 2px 4px rgba(0, 0, 0, 0.14), 0 3px 4px rgba(0, 0, 0, 0.12), 0 1px 5px rgba(0, 0, 0, 0.2);
 
-  &.loading {
-    &::before {
-      content: '';
-      position: relative;
-      display: inline-block;
-      width: 45%;
-      height: 1.2em;
-      background-color: theme.$card-loading-bg-color;
-      border-radius: 5px;
-    }
-  }
-
   @include responsive.mobile-only {
     font-size: 14px;
   }
 }
 
+.loading {
+  &::before {
+    content: '';
+    position: relative;
+    display: inline-block;
+    width: 45%;
+    height: 1.2em;
+    background-color: theme.$card-loading-bg-color;
+    border-radius: 5px;
+  }
+}
+
```

Comment:
Is this move needed?

Reply:
Yes! Previously, it was only available in the `.title` css selector. With this change, it can now also be used elsewhere (for example for the scheduled start)